raster/src/main/scala/geotrellis/raster/Raster.scala

```diff
@@ -33,6 +33,30 @@ case class Raster(tile: Tile, extent: Extent) extends Product2[Tile, Extent] {
 
   def asFeature(): Feature[Extent, Tile] = ExtentFeature(extent, tile)
 
+  def get(point: Point): Int =
+    get(point.x, point.y)
+
+  def get(col: Int, row: Int): Int =
+    tile.get(col, row)
+
+  def get(x: Double, y: Double): Int =
```

Comment:
These two `get` methods should not have the same name. They do different things and with automatic upgrades from Int to Double there may be surprising results. I've had this happen with another `geotrellis.raster` class and it wasn't great. Maybe `getPointValue` and `getDoublePointValue`.